com.github.sdbg.debug.ui/src/com/github/sdbg/ui/launchers/IChromeLaunchShortcutExt.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui.launchers;
+
+import org.eclipse.core.resources.IResource;
+
+public interface IChromeLaunchShortcutExt {
```

Comment:
This would also fix the build issue around having to know about SDBG classes (and yet another build dependency to download). :-)